MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+             x:Class="ListViewPrism.Views.MainPage"
+             Title="MainPage">
+    <StackLayout HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand">
+        <syncfusion:SfListView x:Name="listView" ItemSize="70" ItemSpacing="0,0,5,0"
+                             AutoFitMode="Height"
+                             ItemsSource="{Binding ContactsInfo,}" IsStickyHeader="True" 
+                             AllowSwiping="True" IsStickyGroupHeader="True" GroupHeaderSize="50">
+        </syncfusion:SfListView>
+    </StackLayout>
+</ContentPage>    
+{% endhighlight %}
+{% endtabs %}
+
+## Improving ListView performance
+
+The .NET MAUI ListView (SfListView) has been built from the ground up with an optimized view reuse strategy for the best possible performance on the .NET MAUI platform even when loading large data sets. Following techniques are used to improve performance of the `SfListView`:
+
+ * Bind the `ItemsSource` property to an IList<T> collection instead of an IEnumerable<T> collection because IEnumerable<T> collection do not support random access.
+ * The `SfListView` gets refreshed each and every time a new item added into the underlying collection. Because, when adding items at runtime, the `DataSource` gets refreshed. To avoid this behavior, use [BeginInit()](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataSource.DataSource.html#Syncfusion_Maui_DataSource_DataSource_BeginInit) to stop the `RefreshView()` calling in each time, and use [EndInit()](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataSource.DataSource.html#Syncfusion_Maui_DataSource_DataSource_EndInit) to start the `RefreshView()` calling when adding number of finished items.
+ * Avoid loading complex layout in template of listview. For example, loading large-size images or nested containers degrades the scrolling performance. This practice commonly degrades performance in all platforms, and particularly more in Android version API level 19. So, use fewer elements and images with small size and resolution to achieve the maximum performance.
+ * Avoid placing the `SfListView` inside `ScrollView` for the following reasons:
+    * The `SfListView` implement its own scrolling.
+    * The `SfListView` will not receive any gestures as it will be handled by the parent `ScrollView`.
+    * Should define size to the `SfListView` if it loads inside `ScrollView`.
+ * Avoid changing the cell layout based on the `BindingContext`. This incurs large layout and initialization costs.
+ * Implement a model class inherited with `INotifyPropertyChanged` interface to notify the property changes at runtime.
+
+ ## Scrolling ListView without virtualization
+
+`SfListView` allows you to scroll by loading the entire collection of items inside the `ScrollView` and defining the total extend of its container to `HeightRequest` in the [Loaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Loaded) event.

```

Comment:
The `SfListView` allows you to scroll by loading the entire collection of items inside the `ScrollView` and defining the total extend of its container to `HeightRequest` in the [Loaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Loaded) event.

Reply:
I have modified the line as per the concerns provided